### ISSUE TITLE
Fails to load

### ISSUE THREAD

**Issue by geoff-m** [ltdrdata/ComfyUI-Impact-Pack#488]
Same problem regardless of whether I install manually with `git clone` or through ComfyUI-Manager GUI.

`No module named 'ultralytics'` ... Full log: [comfyui.log](https://github.com/ltdrdata/ComfyUI-Impact-Pack/files/14402792/comfyui.log)

No ComfyUI extensions are installed other than ComfyUI-Manager.

I did not encounter this problem when using ComfyUI's Windows standalone release.

**Comment by ltdrdata** [comment]
Show me the result of this.

`C:\Users\Geoff\miniconda3\python.exe -m pip freeze`

**Comment by geoff-m** [comment]
Sorry, I uninstalled miniconda after making this report. The issue persists but the message is now `No module named 'safetensors'` ([full log](https://github.com/ltdrdata/ComfyUI-Impact-Pack/files/14458278/comfyui.log))

The output of `python -m pip freeze` is now [this](https://github.com/ltdrdata/ComfyUI-Impact-Pack/files/14458291/freeze.txt).

**Comment by ltdrdata** [comment]
> Sorry, I uninstalled miniconda after making this report. The issue persists but the message is now `No module named 'safetensors'` ([full log](https://github.com/ltdrdata/ComfyUI-Impact-Pack/files/14458278/comfyui.log))
> 
> The output of `python -m pip freeze` is now [this](https://github.com/ltdrdata/ComfyUI-Impact-Pack/files/14458291/freeze.txt).

Is there a reason you're not using the portable version in a Windows environment? 
I personally set up and use a venv environment. In this case, I have to manually install torch and others as described in the installation section of ComfyUI.

https://github.com/comfyanonymous/ComfyUI#manual-install-windows-linux

**Comment by geoff-m** [comment]
> Is there a reason you're not using the portable version in a Windows environment? 

Yes, I am trying to get the git version to work to avoid a bug I experienced with ComfyUI-Manager. I have used the standalone version, but the issue I'm reporting here only occurs when using the git version. Does ComfyUI-Impact-Pack not support the git version of ComfyUI?

**Comment by ltdrdata** [comment]
> > Is there a reason you're not using the portable version in a Windows environment?
> 
> Yes, I am trying to get the git version to work to avoid a bug I experienced with ComfyUI-Manager. I have used the standalone version, but the issue I'm reporting here only occurs when using the git version. Does ComfyUI-Impact-Pack not support the git version of ComfyUI?

For some reason, it appears that your package installation list indicates that the install script didn't run properly. You need to install the requirements.txt in your ComfyUI execution Python environment.

Otherwise, it seems like you might be confusing the Python environment where ComfyUI is currently running.
